yatdlm/todo/models.py

```diff
@@ -250,6 +267,14 @@ def get_categories(self):
 
         return [cat.as_dict() for cat in categories]
 
+    def set_categories(self, categories_ids: T.List[int]) -> None:
+        if not Category.objects.filter(id__in=categories_ids).exists():
+            return
+
+        self.categories.clear()
+
+        self.categories.add(*categories_ids)
```

Comment:
Seems to me this could 'just be' `self.categories = categories_ids` but I'm not familiar with Django's ORM.